src/_interfaces/injector.js

```diff
@@ -0,0 +1,26 @@
+// @flow
```

Comment:
are these just pre-compile files for flow? Can we not combine all interfaces into one file?